### DISCUSSION TITLE
Brand Name Search Filter

### DISCUSSION THREAD

**Discussion by vakit** [inspireui/support#5567] [New Feature/Ideas Requests]
Hi. 
I just want to tell you for new feature. 

We have brand name section for products. For example we can add brands like Nike, Adidas... but users can't search products with brand name. if we add brand name search filter I think this will be great. 

Thanks

**Comment by osman-kocak** [comment]
new ajax search plugin can do it

**Reply by vakit** [reply]
Which plugin? You mean I need to install new plugin? Could you send me link of plugin? 
Thanks

**Reply by osman-kocak** [reply]
Ajax Search Pro
